src/lib/tsc-watch.ts

```diff
@@ -71,6 +75,27 @@ function killProcesses(currentCompilationId: number, killAll: boolean): Promise<
   return runningKillProcessesPromise;
 }
 
+let runningKillEmitProcessesPromise: Promise<number> | null = null;
+function killEmitProcesses(currentEmitId: number): Promise<number> {
```

Comment:
I'm not sure that I understand this function... can you please explain?

Reply:
Just added a comment to the function